### ISSUE TITLE
Misnamed feature gate in API server help

### ISSUE THREAD

**Issue by sftim** [kubernetes/kubernetes#101309]
#### What happened:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#list-deployment-v1-apps (which is generated automatically) mentions a feature gate named `WatchBookmarks`. This is not the correct name; that feature gate is / was `WatchBookmark`. Also, the feature has graduated.

So what happened was I saw an incorrect feature gate name.

#### What you expected to happen:
The generated docs don't mention `WatchBookmark` _or_ `WatchBookmarks` 

ie, I think the fix is to remove
```go
	// If the feature gate WatchBookmarks is not enabled in apiserver,
	// this field is ignored.
```

#### How to reproduce it (as minimally and precisely as possible):
- generate the documentation
- read the documentation

or, short cut, go and look at https://github.com/kubernetes/apimachinery/blob/8daf28983e6ecf28bd8271925ee135c1179ad13a/pkg/apis/meta/v1/types.go#L360

#### Environment:
- Kubernetes version (use `kubectl version`): v1.21
- Cloud provider or hardware configuration: n/a
- OS (e.g: `cat /etc/os-release`): n/a
- Kernel (e.g. `uname -a`): n/a
- Install tools: n/a
- Network plugin and version (if this is a network-related bug):
- Others: 

/kind bug
/sig api-machinery
/priority backlog
(IMO)

**Comment by sftim** [comment]
This issue replaces https://github.com/kubernetes/website/issues/27623

**Comment by dims** [comment]
cc @ykakarap @palnabarun

**Comment by sftim** [comment]
This could count as a good-first-issue?

**Comment by palnabarun** [comment]
I saw that the feature gate and the comment were added in https://github.com/kubernetes/kubernetes/commit/2a76b82394a8853298f64bfe83df1015a05259e6. I agree that the sentence about the feature gate can be removed from the description since the feature moved to GA.

> This could count as a good-first-issue?

I think it can.

/good-first-issue
/help-wanted

(Applying the labels for now. Leaving the triage bit for the recurring triage sessions)
